db/python/tables/analysis.py

```diff
@@ -502,14 +503,18 @@ async def get_analysis_runner_log(
             values['project_ids'] = project_ids
 
         if author:
-            wheres.append('audit_log_id = :audit_log_id')
-            values['audit_log_id'] = await self.audit_log_id()
+            wheres.append('author = :author')
+            values['author'] = author
```

Comment:
I don't think we can filter this way for new ar-entries after the audit_log migration.
I think (a bit awkward), but we have to do some lookup on the audit_log for the new ones, and maybe more strongly consider some migration to unify this.

Noting that the UI is actually wrong too, it should look at the `on_behalf_of` column in the audit log.